src/main/java/com/arpnetworking/tsdcore/sinks/AggregationServerHttpSink.java

```diff
@@ -76,6 +76,7 @@ protected String getContentType() {
                 .setMetric(metricName)
                 .setPeriod(periodicData.getPeriod().toString())
                 .setPeriodStart(periodicData.getStart().toString())
+                .setMinRequestTime(periodicData.getMinRequestTime().toString())
```

Comment:
You will then need to deal with the Optionality of the field here and in the `AggregationServerSink` as well; maybe use `ifPresent`.